src/frontend/styles/components/sidebar.pcss

```diff
@@ -1,20 +1,20 @@
 .docs-sidebar {
   width: 100vw;
-    
+
   @media (--desktop) {
+    position: fixed;
```

Comment:
I wouldn't recommend using fixed positioning here as it leads to content overlapping sidebar on narrow screens

https://user-images.githubusercontent.com/31101125/174969478-14d86f08-417e-4587-9a3e-e915b8dbc82f.mov

I suggest leaving `position: sticky` for `.docs-sidebar__content` instead